automation/project/Dependencies.scala

```diff
@@ -5,7 +5,7 @@ object Dependencies {
   val akkaV = "2.5.7"
   val akkaHttpV = "10.0.10"
 
-  val serviceTestV = "0.1-a45247c-SNAP"
+  val serviceTestV = "0.1-5bf255f-SNAP"
```

Comment:
NOTE ... this is not a permanent version number. When the associated workbench-lib PR merges, I'll need to grab the latest version.